crmsh/qdevice.py

```diff
@@ -173,20 +245,24 @@ def valid_qnetd(self):
         """
         Validate on qnetd node
         """
-        if utils.check_ssh_passwd_need(self.qnetd_addr):
-            self.askpass = True
-
         exception_msg = ""
         suggest = ""
         if utils.service_is_active("pacemaker", self.qnetd_addr):
-            exception_msg = "host for qnetd must be a non-cluster node"
+            exception_msg = "Host for qnetd must be a non-cluster node!"
             suggest = "change to another host or stop cluster service on {}".format(self.qnetd_addr)
         elif not utils.package_is_installed("corosync-qnetd", self.qnetd_addr):
-            exception_msg = "Package \"corosync-qnetd\" not installed on {}".format(self.qnetd_addr)
+            exception_msg = "Package \"corosync-qnetd\" not installed on {}!".format(self.qnetd_addr)
             suggest = "install \"corosync-qnetd\" on {}".format(self.qnetd_addr)
-
+        if utils.service_is_active("corosync-qnetd", self.qnetd_addr):
+            cmd = "corosync-qnetd-tool -l -c {}".format(self.cluster_name)
+            if utils.get_stdout_or_raise_error(cmd, remote=self.qnetd_addr):
+                exception_msg = "This cluster's name \"{}\" already exists on qnetd server!".format(self.cluster_name)
+                suggest = "consider to use the different cluster-name property"
```

Comment:
Revisit this. I feel the information of ERROR is good enough, even better to delete above line 
```suggestion
```

Reply:
It's consistent style for the previous checking information, like:
```
            exception_msg = "Host for qnetd must be a non-cluster node!"
            suggest = "change to another host or stop cluster service on {}".format(self.qnetd_addr)
```
```
            exception_msg = "Host for qnetd must be a non-cluster node!"
            suggest = "change to another host or stop cluster service on {}".format(self.qnetd_addr)
```
The output will be:
```
ERROR: cluster.init: Host for qnetd must be a non-cluster node!
Cluster service already successfully started on this node except qdevice service.
If you still want to use qdevice, change to another host or stop cluster service on hanode2.
Then run command "crm cluster init" with "qdevice" stage, like:
        crm cluster init qdevice qdevice_related_options
That command will setup qdevice separately.
```
```
ERROR: cluster.init: Package "corosync-qnetd" not installed on hanode2!
Cluster service already successfully started on this node except qdevice service.
If you still want to use qdevice, install "corosync-qnetd" on hanode2.
Then run command "crm cluster init" with "qdevice" stage, like:
        crm cluster init qdevice qdevice_related_options
That command will setup qdevice separately.
```

Do you think the suggest information is also unnecessary for previous cases? Just like:
```
ERROR: cluster.init: Host for qnetd must be a non-cluster node!
```
```
ERROR: cluster.init: Package "corosync-qnetd" not installed on hanode2!
```
are good enough?

Reply:
I like the pattern of "msg/suggestion" very much, especially the suggestion part !  Just in some simple case, msg part is self-explained, then no necessarily to provide the suggestion. 

For the additional two errors:

> ERROR: cluster.init: Package "corosync-qnetd" not installed on hanode2!
The corresponding "suggestion" does make sense to me.

> ERROR: cluster.init: Host for qnetd must be a non-cluster node!
However, this ERROR is too strong in the bigger picture. It does not fit the situation to add HA capability to the qnetd server itself. With that in mind, I do suggest to delete this check, the corresponding ERROR and suggestion.

Reply:
Changed